roles/lib_utils/action_plugins/master_check_paths_in_config.py

```diff
@@ -32,6 +32,7 @@
     '/etc/origin/master/',
     '/var/lib/origin',
     '/etc/origin/cloudprovider',
+    '/etc/origin/kubelet-plugins',
```

Comment:
If this is hardcoded here then it shouldn't be a way to override `openshift_flexvolume_container_directory_default`

Reply:
Yes that is true but since control plane components are being run as static pods now, users can still override it by editing pod spec. They just can't use ansible to do that without changing this value as well.

Reply:
> since control plane components are being run as static pods now, users can still override it by editing pod spec

Correct, we can't prevent that, all we care about is ansible options - currently user can override `openshift_flexvolume_container_directory_default` and get a broken install. Its okay to hardcode this path everywhere - and document it properly

Reply:
We merged https://github.com/openshift/openshift-ansible/pull/9046 to remove configuration of this property. It is no longer possible to configure this via Ansible. On Atomic hosts, the location has to be `/etc/origin/kubelet-plugins` anyways because of the way image is built. So, it made little sense to allow user to configure this in >=3.10